packages/js/src/consts/index.ts

```diff
@@ -0,0 +1,3 @@
+export const CONTENT_TYPE = {
+  FORM_URL_ENCODED: { 'Content-Type': 'application/x-www-form-urlencoded' },
```

Comment:
we don't have a convention of using upper snake case for constants. Just use camelCase for the key and PascalCase for the object name is fine.

Reply:
done